### ISSUE TITLE
Unit test for header - profile name displays

### ISSUE THREAD

**Issue by Sheeri** [WildAid/o-fish-web#316]
**Is your feature request related to a problem? Please describe.**
We would like to start adding unit tests for components in our app.

**Describe the solution you'd like**
A test in the directory src/components/__test__ (creating the dir if it doesn't already exist) that tests the part of header/header.component.js which displays the profile name. (class name is 'profile-name').

**Comment by mitrabuggaveeti** [comment]
Would like to work on this !!!!!

**Comment by Sheeri** [comment]
Hello @mitrabuggaveeti, thanks for offering to work on this! I have assigned it to you.

**Comment by Sheeri** [comment]
Hi @mitrabuggaveeti are you still working on this?

**Comment by mitrabuggaveeti** [comment]
Sorry ,was busy this week . Assign it to someone..

**Comment by Sheeri** [comment]
No worries, I just noticed during office hours and remembered our great convo last week in office hours. I'm unassigning, but nobody is chomping at the bit for this task just yet, so if you get around to it, we can re-assign to you. We totally understand that life gets in the way!

**Comment by Kat-Codes** [comment]
Hey there, I'm interested in working on this 😄

**Comment by Sheeri** [comment]
Hello @Kat-Codes, thanks for offering to work on this! I have assigned it to you.

**Comment by lenmorld** [comment]
Hi @Sheeri 
Is this still available?
I'd like to work on this one since I ventured with tests already in [317](https://github.com/WildAid/o-fish-web/issues/317)

**Comment by Sheeri** [comment]
Go for it! FWIW feel free to just make tests and if there aren't issues for the tests you're making, you can make an issue for it!

**Comment by lenmorld** [comment]
@Sheeri I'm just coming back to open source after a long break. I'm ready again :) 
Just want to make sure, is this project still active?
I can work on this one to start, sorry I didn't realize it was assigned to me all this time

**Comment by Sheeri** [comment]
Yes, no worries, nobody else has wanted to work on it yet, so you didn't block anyone.
